pkg/commands/options/build.go

```diff
@@ -67,6 +67,9 @@ type BuildOptions struct {
 
 	// BuildConfigs stores the per-image build config from `.ko.yaml`.
 	BuildConfigs map[string]build.Config
+
+	// If true, don't convert Docker-typed base images to OCI when building.
+	PreserveDockerMediaType bool
```

Comment:
Wdyt about `PreserveMediaTypes` instead? It's not necessarily docker media types being preserved.

Reply:
Isn't it? The conversion only goes the one way, from docker media types to OCI types. Are there other types you're thinking of?

Reply:
Not specifically but it's a shorter flag without `docker` and the code just preserves the existing media type rather than only docker media types.

Reply:
Fair enough. Change incoming.